test_wrappers.py

```
@@ -78,17 +80,24 @@ def _run(wrapper, cmd, check_log=None, compare_results_with_expected=None):
         testdir = os.path.join(tmp_test_subdir, "test")
         shutil.copytree(os.path.join(wrapper, "test"), testdir)
 
+        breakpoint()
```

Ruff (0.14.3)

Trace found: `breakpoint` used

(T100)